.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!--
+  Describe what changes are being made, e.g. which feature/bug is being
+  developed/fixed in this PR?
+-->
+
+**Checklist**:
+
+<!-- add "N/A" to the end of each line not applicable to your changes -->
+
+<!-- to check an item, place an "x" in the box like so: "- [x] Tests" -->
+
+- [ ] Tests
+- [ ] [CHANGELOG](https://github.com/greenbone/gsa/blob/master/CHANGELOG.md) Entry
```

Comment:
Not sure if you have a CHANGELOG.md in gmvd yet.
```suggestion
- [ ] [CHANGELOG](https://github.com/greenbone/gvmd/blob/master/CHANGELOG.md) Entry
```

Reply:
Yes, we have one. I just forgot to update the path in the template.